RELEASE_NOTES.md

```diff
@@ -4,26 +4,135 @@
+
+New cookbooks containing the same functionality:
+```ruby
+  include_recipe 'yum-epel'
+  include_recipe 'yum-ius'
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/DefaultMetadataMaintainer
 
+The `DefaultMetadataMaintainer` cop detects the default `maintainer` and `maintainer_email` fields generated by the `chef generate cookbook` command. Cookbook metadata should be updated with actual maintainer information.
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/UsesDeprecatedMixins
 
+The `UsesDeprecatedMixins` cop detects various legacy mixins used in `HWRP` and `LWRP` resources. These mixins are no longer necessary and were removed in Chef Infra Client 14, which will cause errors running these resources on modern Chef Infra Client releases.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
 #### Chef/MinitestHandlerUsage
 
+The `MinitestHandlerUsage` cop detects cookbooks that depend on the [minitest-handler](https://supermarket.chef.io/cookbooks/minitest-handler) cookbook. This cookbook provides a handler for running [minitests](https://github.com/seattlerb/minitest) after each Chef Infra Client run. The handler is no longer maintained and we highly recommend using InSpec for validating system configuration post Chef Infra Client run. InSpec includes over 100 build in resources and integrates with Automate to report the results of system state validation.
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/WindowsVersionHelper
 
+The `WindowsVersionHelper` cop detects the usage of the various Windows version helpers that shipped in the [windows](https://supermarket.chef.io/cookbooks/windows) cookbook. Nearly all of the resources that originally shipped in Windows cookbook are now part of Chef Infra Client itself. We'd like to help move users away from this cookbook so they can avoid the added dependency in their environment. Instead of using the Windows version helpers we suggest using data from Ohai itself to determine what version of Windows is running on a node.
+
+`Examples`
+
+A cookbook using the Windows version helper:
+```ruby
+  Windows::VersionHelper.nt_version == 6.1
+```
+
+The same version comparison could be done using Ohai attributes
+```ruby
+  node['platform_version'].to_f == 6.1
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/WindowsZipfileUsage
 
+The `WindowsZipfileUsage` detects the usage of the `windows_zipfile` resource in the [windows](https://supermarket.chef.io/cookbooks/windows) cookbook. This is one of the remaining resources in the windows cookbook that is not present in Chef Infra Client itself. In Chef Infra Client 15.0 we introduced the [archive_file](https://docs.chef.io/resource_archive_file.html) resource, which is a cross platform resource for uncompressing archives. This resource uses an ulta-fast compression library with support for multiple compression formats, and should be used instead of the legacy `windows_zipfile` resource.
```

Comment:
```suggestion
The `WindowsZipfileUsage` detects the usage of the `windows_zipfile` resource in the [windows](https://supermarket.chef.io/cookbooks/windows) cookbook. The `windows_zipfile` resource is one of the remaining resources in the windows cookbook that is not present in Chef Infra Client itself. In Chef Infra Client 15.0, we introduced the [archive_file](https://docs.chef.io/resource_archive_file.html) resource, which is a cross-platform resource for uncompressing archives. This new resource uses an ultra-fast compression library with support for multiple compression formats, and should be used instead of the legacy `windows_zipfile` resource.
```